packages/zoe/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.json",
-  "compilerOptions": {},
+  "compilerOptions": {"skipLibCheck": false},
```

Comment:
help me understand this `skipLibCheck` change?

Reply:
Oh that was to get a more rigorous check while debugging. I'll remove it.

https://www.typescriptlang.org/tsconfig/#skipLibCheck